kakao/src/main/kotlin/com/agoda/kakao/list/KListView.kt

```diff
@@ -99,12 +100,12 @@ class KListView : ScrollViewActions, BaseAssertions, ListViewAdapterAssertions {
             throw IllegalStateException("${T::class.java.simpleName} did not register to KListView")
         }.provideItem
 
-        val interaction = Espresso.onData(Matchers.anything())
+        val interaction = DataInteractionDelegate(Espresso.onData(Matchers.anything()))
```

Comment:
There is no need for data interaction delegate, since there is no interceptable call